node-w3capi/markdown.js

```diff
@@ -0,0 +1,14 @@
+import tablemark from 'tablemark'
```

Comment:
likewise, a markdown table seems a very easy thing to construct, do we need a library that then we have to keep up to date just for this simple thing?

Reply:
We don't need it keep it up to date if we are not going to use that code anytime soon.
Please feel free to write that table generation code yourself if you don't want to take advantage of an existing npm module.

Reply:
Can we output HTML instead and reuse the HTML generating code?